### ISSUE TITLE
Icon Renaming: git icons

### ISSUE THREAD

**Issue by SirWindfield** [feathericons/feather#737]
I would like to propose a renaming of the `git` icons. These icons can be used in conjunction with any version control system. What about renaming them into `code-branch`, `code-commit`, `code-pull-request` etc.?

**Comment by locness3** [comment]
I don't know...

**Comment by SirWindfield** [comment]
Well, you could provide some points against it if you don't agree with it. I am fine with either feedback. It's just that I think that they can be used in broader terms and not only in conjunction with git.

**Comment by ahtohbi4** [comment]
If we start to develop your thought, we come to an idea that we can store not only code in CVS.

I suppose that `git`, in this case, is a well-known and easily recognized abstraction.

**Comment by SirWindfield** [comment]
No, it's not that we cannot only store code. I was thinking about other VCS. svn, git, mercurial, bazaar. There are a lot of them.

**Comment by MarcelloTheArcane** [comment]
Can the search on the website have synonyms? If so, I support renaming to code and adding the synonym of git.
